src/test/java/TC_DP194_51.java

```diff
@@ -0,0 +1,30 @@
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+import steps.CategoryPageStep;
+import steps.CheckoutPageAsLoggedUserStep;
+import steps.HomePageStep;
+
+public class TC_DP194_51 extends CommonConditionTest{
+
+    @ParameterizedTest
+    @CsvFileSource(resources = "/LoginValuesCheckoutPage.csv",numLinesToSkip = 1)
+    public void checkoutAsLoggedUser(String email, String password){
+
+        String contactPageUrl = "http://34.121.117.87/index.php?route=checkout/success";
```

Comment:
We use as an expected URL constants in PageURL interface (constants package). You should have it (package) if you pulled last updates from div branch